src/lua.rs

```diff
@@ -625,6 +626,69 @@ impl Lua {
         }
     }
 
+    /// Set a function for Lua to call on conditions configured by `options`. This function will
+    /// always succeed, but the hook may raise an error to be reported back to the caller when
+    /// running Lua code.
+    ///
+    /// # Example
+    ///
+    /// Shows each line number of code being executed by the Lua interpreter.
+    ///
+    /// ```
+    /// # extern crate rlua;
+    /// # use rlua::{Lua, HookOptions, Debug};
+    /// # fn main() {
+    /// let code = r#"local x = 2 + 3
```

Comment:
Ultra-nitpick: try putting a newline after the `r#"` and indent the Lua code to make it look a bit nicer

Reply:
I did not since Lua would report it being on the 2nd line (2-3-4 would be outputted) and that looked confusing to me. Might change it to something else.